src/main/kotlin/com/redhat/devtools/intellij/kubernetes/tree/KubernetesStructure.kt

```diff
@@ -82,403 +85,440 @@ class KubernetesStructure(model: IResourceModel) : AbstractTreeStructureContribu
-			val CRONJOBS = Folder("CronJobs", CronJobsOperator.KIND) //  Workloads / CronJobs
-            val PODS = Folder("Pods", NamespacedPodsOperator.KIND) //  Workloads / Pods
+        val DEPLOYMENTS = Folder("Deployments", DeploymentsOperator.KIND) //  Workloads / Deployments
+        val STATEFULSETS = Folder("StatefulSets", StatefulSetsOperator.KIND) //  Workloads / StatefulSets
+        val DAEMONSETS = Folder("DaemonSets", DaemonSetsOperator.KIND) //  Workloads / DaemonSets
+        val JOBS = Folder("Jobs", JobsOperator.KIND) //  Workloads / Jobs
+        val CRONJOBS = Folder("CronJobs", CronJobsOperator.KIND) //  Workloads / CronJobs
+        val PODS = Folder("Pods", NamespacedPodsOperator.KIND) //  Workloads / Pods
         val NETWORK = Folder("Network", null)
-            val SERVICES = Folder("Services", ServicesOperator.KIND) // Network / Services
-            val ENDPOINTS = Folder("Endpoints", EndpointsOperator.KIND) // Network / Endpoints
-			val INGRESS = Folder("Ingress", IngressOperator.KIND) // Network / Ingress
-		val STORAGE = Folder("Storage", null)
-			val PERSISTENT_VOLUMES = Folder("Persistent Volumes", PersistentVolumesOperator.KIND) // Storage / Persistent Volumes
-			val PERSISTENT_VOLUME_CLAIMS = Folder("Persistent Volume Claims", PersistentVolumeClaimsOperator.KIND) // Storage / Persistent Volume Claims
-			val STORAGE_CLASSES = Folder("Storage Classes", StorageClassesOperator.KIND) // Storage / Storage Classes
-		val CONFIGURATION = Folder("Configuration", null)
-			val CONFIG_MAPS = Folder("Config Maps", ConfigMapsOperator.KIND) // Configuration / Config Maps
-			val SECRETS = Folder("Secrets", SecretsOperator.KIND) // Configuration / Secrets
-		val CUSTOM_RESOURCES_DEFINITIONS = Folder("Custom Resources", CustomResourceDefinitionsOperator.KIND)
+        val SERVICES = Folder("Services", ServicesOperator.KIND) // Network / Services
+        val ENDPOINTS = Folder("Endpoints", EndpointsOperator.KIND) // Network / Endpoints
+        val INGRESS = Folder("Ingress", IngressOperator.KIND) // Network / Ingress
+        val STORAGE = Folder("Storage", null)
+        val PERSISTENT_VOLUMES =
+            Folder("Persistent Volumes", PersistentVolumesOperator.KIND) // Storage / Persistent Volumes
+        val PERSISTENT_VOLUME_CLAIMS = Folder(
+            "Persistent Volume Claims",
+            PersistentVolumeClaimsOperator.KIND
+        ) // Storage / Persistent Volume Claims
+        val STORAGE_CLASSES = Folder("Storage Classes", StorageClassesOperator.KIND) // Storage / Storage Classes
+        val CONFIGURATION = Folder("Configuration", null)
+        val CONFIG_MAPS = Folder("Config Maps", ConfigMapsOperator.KIND) // Configuration / Config Maps
+        val SECRETS = Folder("Secrets", SecretsOperator.KIND) // Configuration / Secrets
+        val CUSTOM_RESOURCES_DEFINITIONS = Folder("Custom Resources", CustomResourceDefinitionsOperator.KIND)
+        val HELM_RELEASES = Folder("Helm Releases", HelmRelease.KIND)
```

Comment:
This reformatting is dropping the indentation that helped to understand the structure. Ex. `INGRESS` is a subnode to `ENDPOINTS`. Furthermore these changes that only affect the formatting add noise that make identifying the real changes difficult.
I'd undo these in the places where they were added if possible.